### PR TITLE
PS-4680 PS & PXC - Document the cnf file locations and best practices.

### PR DESCRIPTION
**Pull request by patrickbirch**
merged with 5.6 version
fixed conflicts in apt_repo, yum_repo
added .res files
added .res file to the conf.py

### CODE REVIEW

**Comment by patrickbirch** [comment]
waiting for comments or approval.

File: doc/source/installation/apt_repo.rst
Comment (tplavcic) [inline doc/source/installation/apt_repo.rst]: Maybe put Ubuntu 16.04 (Xenial) instead Vivid since Vivid is no longer supported and was not LTS release anyway.

**Review by tplavcic — APPROVED** [review]

File: doc/source/installation/apt_repo.rst
Comment (patrickbirch) [reply doc/source/installation/apt_repo.rst]: updated to 16.04

**Pull request merged**
